frontend/src/CreateEvent/CreateEvent.tsx

```diff
@@ -0,0 +1,54 @@
+import classes from "./CreateEvent.module.css";
+import Button, { ButtonOptions } from "../Button/Button";
+import { ChevronLeftIcon } from "@heroicons/react/24/solid";
+import { CameraIcon } from "@heroicons/react/24/outline";
+
+function CreateEvent() {
+  return (
+    <div className={classes.container}>
+      <button className={classes.back}>
```

Comment:
if you could make this and its corresponding style its own component that would be really appreciated!